Pipelines/Scripts/GetNugetVersion.ps1

```diff
@@ -34,20 +34,38 @@ if(([regex]::Matches($VersionFormat, "\." )).count -ne 2)
 
 $BuildId = $env:BUILD_BUILDID
 Write-Host "BuildId:" $BuildId
+$FullBranchName = $env:BUILD_SOURCEBRANCH
+$branchName = $FullBranchName.Replace("refs/heads/", "")
 
-if ($env:releaseId)
+if ($branchName -like 'release*')
 {
-    Write-Host "Building for release" $env:releaseId
+    Write-Host "Building for release"
     $Suffix = ""
 }
 else
 {
-    $Branch = $env:BUILD_SOURCEBRANCHNAME
-    Write-Host "Branch:" $Branch
-    $Suffix = if ($Branch -eq "master") {"-beta"} else {"-alpha"}	
+    Write-Host "Branch:" $branchName
+    $Suffix = if ($branchName -like 'master') {'-beta'} else {'-alpha'}	
     Write-Host "Suffix:" $Suffix
 }
 
+$releaseName = $branchName.Split("/");
```

Comment:
You may give an example of what you are expecting here in order to make it easier to understand (like release/2023-P3S5). 
In the else clause, do you expect it to be equal to 0, 1 or more than 2? Can you give an example of what is expected there?